src/GRA.Data/Repository/PsBranchSelectionRepository.cs

```diff
@@ -49,7 +49,9 @@ public async Task<int> GetCountByPerformerIdAsync(int performerId)
         {
             return await DbSet
                 .AsNoTracking()
-                .Where(_ => _.ProgramId.HasValue && _.Program.PerformerId == performerId)
+                .Where(_ => _.ProgramId.HasValue
```

Comment:
Move the Where clause into the `.CountAsync()`

Reply:
Fixed in 7ea0e75f2f09c9275a89a5a37a81ef42cf2b2fba